extensions/default/src/getHangingProtocolModule.js

```diff
@@ -14,6 +14,7 @@ const defaultProtocol = {
   editableBy: {},
   protocolMatchingRules: [],
   toolGroupIds: ['default'],
+  minSeriesLoadedToRunHP: 10,
```

Comment:
This shouldn't be a simple value, but should be something like:
runHPCondition: {
  minSeriesLoaded: 10,
}
then, that condition can have other child elements added in the future to define more conditions.